generate.pl

```diff
@@ -309,7 +326,12 @@ =head1 DESCRIPTION
     && curl -fLO {{cpanm_dist_url}} \
     && echo '{{cpanm_dist_sha256}} *{{cpanm_dist_name}}.tar.gz' | sha256sum --strict --check - \
     && tar -xzf {{cpanm_dist_name}}.tar.gz && cd {{cpanm_dist_name}} && perl bin/cpanm . && cd /root \
-    && cpanm IO::Socket::SSL \
+    && curl -fLO '{{netssleay_dist_url}}' \
+    && echo '{{netssleay_dist_sha256}} *{{netssleay_dist_name}}.tar.gz' | sha256sum --strict --check - \
+    && cpanm --from $PWD {{netssleay_dist_name}}.tar.gz \
+    && curl -fLO '{{iosocketssl_dist_url}}' \
+    && echo '{{iosocketssl_dist_sha256}} *{{iosocketssl_dist_name}}.tar.gz' | sha256sum --strict --check - \
+    && SSL_CERT_DIR=/etc/ssl/certs cpanm --from $PWD {{iosocketssl_dist_name}}.tar.gz \
```

Comment:
Setting `SSL_CERT_DIR` here is a workaround for https://github.com/noxxi/p5-io-socket-ssl/pull/154 which is part of the reason I started looking at this. It turns out this was pulling in Mozilla::CA when it wasn't actually used, so the extra set of CA certs was in the image, doing nothing.

By telling it to use the system CA certs, Makefile.PL sees that and doesn't install Mozilla::CA (which isn't used later, anyway, as the codepaths differed, per the other PR).

Reply:
Good catch on Mozilla::CA going unused - and as the base image for this does get updated regularly, we can ensure freshness on ca-certificates at least for the support window of the Perl versions we publish.